tools/ds-identify

```diff
@@ -590,8 +590,8 @@ read_datasource_list() {
     case "$DI_KERNEL_CMDLINE" in
         *cc:*datasource_list*)
             t=${DI_KERNEL_CMDLINE##*datasource_list}
-            t=${t%%$cb*}
-            t=${t##*$ob}
+            t=${t%%"$cb"*}
+            t=${t##*"$ob"}
```

Comment:
Stylewise, let's we make this even more conspicuous that we are expanding the cb/ob varianbles with curly braces
```suggestion
            t=${t%%"${cb}"*}
            t=${t##*"${ob}"}
```